test/db/cmd/lea_intel

```diff
@@ -34,9 +34,9 @@ EXPECT=<<EOF
   0x00402cb0      call sym.imp.memcmp
   0x00402cb5      test eax, eax
   0x00402cb7      je 0x402cc2
-  0x00402cb9      mov byte [obj.DrawGoodWork], 0                       ; .//ezpz.cpp:270
+  0x00402cb9      mov byte [obj.DrawGoodWork], 0                       ; /media/sf_I_DRIVE/code/gynvael/DragonSector/ds_ctf_main_2016/main-ezpz/src/ezpz.cpp:270
```

Comment:
full path of dwarf files is sometimes a pain, there must be an option to just show the relative filename starting at a specific subdirectory or just the filename like it was done before

Reply:
>  like it was done before

Only reason it showed only filename before was because the parsing didn't work properly and it didn't find the comp_directory. I can remove it and show just the filename I guess, I am not familiar with the line information processing outside the dwarf scope.

This is the part that is doing that I think, and I am not sure what purpose it really serves now, because now we compile all the units and not just the first one, that means that there will be multiple comp_dirs to save and we only save the last one because it keeps overwritting and previous comp_dir
```c
	// TODO  does this have a purpose anymore?
	// Or atleast it needs to rework becase there will be
	// more comp units -> more comp dirs and only the last one will be kept
	if (attribute->attr_name == DW_AT_comp_dir && is_valid_string_form) {
		const char *name = attribute->string.content;
		sdb_set (sdb, "DW_AT_comp_dir", name, 0);
	}
```

Reply:
In some cases `r_bin_addr2text` is not honoring the `asm.dwarf.abspath` variable.

Replace https://github.com/radareorg/radare2/blob/bd9a604a87c59051e0464e961c2e4ac89f6d1415/libr/bin/dbginfo.c#L81 with:
```C
			if (origin > 1) {
				file_nopath = file;
			} else {
				file_nopath = strrchr (file, '/');
				if (file_nopath) {
					file_nopath++;
				} else {
					file_nopath = file;
				}
			}
			return r_str_newf ("%s:%d", file_nopath? file_nopath: "", line);
```

Reply:
yep, that solved it, thanks!